apps/dotcom/zero-cache/package.json

```diff
@@ -11,9 +11,11 @@
 		"defaults"
 	],
 	"scripts": {
-		"dev": "concurrently 'yarn docker-up' 'yarn migrate --signal-success'",
+		"dev": "yarn check-migrations && concurrently 'yarn docker-up' 'yarn migrate --signal-success'",
 		"docker-up": "docker compose --env-file .env -f ./docker/docker-compose.yml up",
+		"docker-up-detached": "docker compose --env-file .env -f ./docker/docker-compose.yml up --detach",
 		"docker-down": "docker compose --env-file .env -f ./docker/docker-compose.yml down",
+		"check-migrations": "yarn docker-up-detached && yarn tsx ./check-migrations.ts && yarn docker-down",
```

Comment:
We run docker in a detached state, that's why we use `docker-down` in the end to make sure we clear it up.